RawSpeed/RawImage.cpp

```diff
@@ -326,7 +326,9 @@ void RawImageData::startWorker(RawImageWorker::RawImageWorkerTask task, bool cro
 void RawImageData::fixBadPixelsThread( int start_y, int end_y )
 {
   int gw = (uncropped_dim.x + 15) / 32;
+#ifdef __AFL_COMPILER
```

Comment:
afl-fuzz works best when the program is very fast, so it can try many samples per second. It's easy to generate a pathological file that has so many bad pixels the run-time goes quadratic. So when compiling under AFL just limit the bad pixels to 100 and bail out after that. It's enough for the code to be tested but doesn't make the fuzzing too slow.

I tried just having a higher limit enabled all the time but the zero_is_bad stuff in RW2 means they actually have a lot of bad pixels detected. Might be a bug in RW2 though.